source/codegen/metadata/nixnet/custom_proto.mako

```diff
@@ -0,0 +1,41 @@
+message FlexRayStats {
+    uint32 NumSyntaxErrorChA = 1;
```

Comment:
All Field names need to be `snake_case`.

Reply:
Changed the field names to snake case